packages/neon_framework/lib/src/models/app_implementation.dart

```diff
@@ -156,25 +161,26 @@ abstract class AppImplementation<T extends Bloc, R extends AppImplementationOpti
   ///
   /// It is mandatory to provide a precompiled SVG under `assets/app.svg.vec`.
   /// SVGs can be precompiled with `https://pub.dev/packages/vector_graphics_compiler`
-  Widget buildIcon({
+  Widget buildIcon(
+    BuildContext context, {
```

Comment:
Still not a fan of passing around the context

Reply:
But using a Builder widget is better? I don't see a problem with passing the context :woman_shrugging:

Reply:
You have no control, where the context is coming from.
A builder also functions as a boundary for rebuilds.